index.html

```diff
@@ -1,10 +1,161 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ru" xmlns="http://www.w3.org/1999/html">
```

Comment:
Bad value http://www.w3.org/1999/html for the attribute xmlns (only http://www.w3.org/1999/xhtml permitted here).
 Please use https://validator.w3.org/
 please delete xmlns="http://www.w3.org/1999/html"